### PR TITLE
network: handle None values of team configuration correctly

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by jkonecny12 — APPROVED** [review]
LGTM!

File: pyanaconda/modules/network/nm_client.py [hunk 1]
```diff
@@ -1639,11 +1639,11 @@ def _update_team_kickstart_network_data(nm_client, iface, connection, network_da
...
-        team_port_cfg = get_team_port_config_from_connection(nm_client, s_uuid)
+        team_port_cfg = get_team_port_config_from_connection(nm_client, s_uuid) or ""
         network_data.teamslaves.append((s_iface, team_port_cfg))
 
     s_team = connection.get_setting_team()
     if s_team:
-        teamconfig = s_team.get_config().replace("\n", "").replace(" ", "")
-    if teamconfig:
-        network_data.teamconfig = teamconfig
+        teamconfig = s_team.get_config() or ""
```
Comment (poncovka) [inline pyanaconda/modules/network/nm_client.py:1647]: You don't have to use `or ""` since you check if `teamconfig` is set in the statement below.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well. :)

File: pyanaconda/modules/network/nm_client.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/modules/network/nm_client.py]: Indeed, updated.

**Pull request merged**